### ISSUE TITLE
Project Progress

### ISSUE THREAD

**Issue by ahkarami** [warmspringwinds/pytorch-cpp#5]
Dear @warmspringwinds,
Your project is fantastic. Do you have nay plan for continuing it? For example by implementing other PyTorch's layers (e.g., nn.Dropout2d, nn.Conv3d)?

**Comment by warmspringwinds** [comment]
@ahkarami 
Thank you.

The models that we are using don't use either dropout of 3d convolution unfortunately
so it's not a direct priority.

Although it's quite easy to do for 3d convolution for example:
https://github.com/warmspringwinds/pytorch-cpp/blob/master/src/pytorch.cpp#L375

You should just use different function instead of ```SpatialDilatedConvolution_updateOutput``` .

If you have interest in these I can guide you on how to do it.

Thank you.

**Comment by ahkarami** [comment]
Dear @warmspringwinds, 
Thank you for your time and response. About implementing 3d convolution and other layers, maybe in the near future I will do that. By the way, I think your project was really fantastic and useful. I hope that you continue to progress it. If you want to know about my reasons, you can see the below link:
[**[Feature Request] Deployment in c++ project**](https://github.com/pytorch/pytorch/issues/3335#issuecomment-370258227)
Best

**Comment by warmspringwinds** [comment]
@ahkarami 
I am glad you liked it :)

Yes, the project is alive an developing -- our focus is just directed towards 2d data so far.
